# Bepaling van warmtecapaciteit van een onbekend materiaal


# Introductie

Onbekende materialen kunnen geïdentificeerd worden door hun eigenschappen te meten. Een van deze eigenschappen is de warmtecapaciteit. In dit practicum gaan we de warmtecapaciteit van een onbekend materiaal bepalen door middel van een calorimeter experiment. Daarbij wordt een bepaalde massa van het materiaal naar een bekende temperatuur gebracht waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaatst. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend.

# Theorie

De soortelijke warmte $c$ van een materiaal is gedefinieerd als de hoeveelheid warmte $Q$ die nodig is om de temperatuur $T$ van een kilogram van het materiaal met één graad Celsius (of één Kelvin) te verhogen:

$$
    c = \frac{Q}{m \Delta T}
$$ (eq:heat_capacity)

Waarbij $Q$ de hoeveelheid warmte in Joules is, $m$ de massa in kilogram is en $\Delta T$ de verandering in temperatuur is. Gegeven de wet van Black, die stelt dat de totale hoeveelheid warmte in een geïsoleerd systeem constant blijft, kunnen we de warmte die het onbekende materiaal verliest gelijkstellen aan de warmte die het water opneemt:

$$
    Q_{materiaal} = -Q_{water} 
$$ (eq:black)

wanneer we de massa's en de begintemperaturen van beide systemen kennen, maar slechts een van de twee soortelijke warmtes, kunnen we de onbekende soortelijke warmte berekenen. We combineren vergelijkingen [](#eq:heat_capacity) en [](#eq:black) om de volgende vergelijking te krijgen:

$$
    T_e = \frac{c_w m_w T_{w,b}+c_m m_m T_{m,b}}{c_w m_w + c_m m_m}
$$ (eq:combined)

Waarbij de subscripts $b$ en $e$ respectievelijk staan voor begintoestand en eindtoestand, $w$ voor water en $m$ voor het onbekende materiaal.

Bij metingen aan verschillende massa's van het onbekende materiaal en vervolgens een least square fit aan bovenstaande vergelijking kunnen we een precieze waarde voor de soortelijke warmte van het onbekende materiaal bepalen. Dat is, wanneer de warmtecapaciteit van bijvoorbeeld de beker te verwaarlozen is.


# Methode en materialen

## Ontwerp
De bovenstaande theorie wordt gebruikt om de soortelijke warmte van een onbekend materiaal te bepalen. Het experiment bestaat uit het verwarmen van verschillende massa's van het onbekende materiaal tot een bekende temperatuur, waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaats. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend. Om de tijd voor het meten van meerdere materialen te reduceren, worden de data van de verschillende groepen in het lokaal samengevoegd. Van tevoren is afgesproken welke massa's door welke groep worden gemeten, en hoeveel water er gebruikt wordt.

## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Calorimeter
- Thermometer of temperatuursensor
- Verwarmingsbron 
- Diverse massablokjes van onbekend materiaal
- Weegschaal
- Water
- Maatcilinder of maatbeker


```{figure} figures/c_onbekend_setup.jpg
:width: 70%
:label: fig_c_onbekend_setup

Een schematische weergave van de opstelling
```


## Procedure
Bespreek wie welke massa's van het onbekende materiaal gaat meten.
Bespreek ook hoeveel water er gebruikt gaat worden.
Bepaal de begintemperaturen.
Hevel het aantal afgesproken massa's in de maatbeker. 
Roer voorzicht zodat de temperatuur homogeen is.
Noteer de hoogste gemeten temperatuur, dit is $T_e$.
Wissel de metingen uit met de andere groepen en voer de data-analyse uit.


```{note}
Hieronder staat een voorbeeld hoe je een grafiek moet opslaan en vervolgens in je document kunt oproepen.
Het voorbeeld kun je natuurlijk straks verwijderen.
```

# Resultaten


In [10]:
m_beker=0.1297 #kg
m_beker_en_water=0.329 #kg
m_beker_en_watern_en_onbekend=0.4305 #kg

v_beker=0.200 #L

T_onbekend_begin=68.9+273.15 #K
T_water_begin=20.9+273.15 #K
T_eind=23.4+273.15 #K

soortwwater=4186 #J/kgK

#Q=c*dm*t

m_onbekend=m_beker_en_watern_en_onbekend-m_beker_en_water #kg
m_water=m_beker_en_water-m_beker #kg

Q=soortwwater*m_water*(T_eind-T_water_begin) #J
c_onbekend=Q/(m_onbekend*(T_onbekend_begin-T_eind)) #J/kgK
verschil_c = Q/(m_onbekend*(T_onbekend_begin-(T_eind+1)))-c_onbekend
percentage_verschil_c = verschil_c/c_onbekend
print(f"De soortelijke warmte van het onbekende materiaal is:{c_onbekend:.2f} J/kgK")
print(f"verschil c per K hoger:{percentage_verschil_c*100:.2f}%")

De soortelijke warmte van het onbekende materiaal is:451.62 J/kgK
verschil c per K hoger:2.25%


```{figure} figures/naam.png
:width: 50%
:label: fig_naam 

Hier is het onderschrift van de figuur.
```

# Discussie en conclusie
De berekende waarde voor warmtecapiciteit is: 451.6 J/kgK. Dit lijkt op die van koper (385 J/kgK), ijzer (460 J/kgK) of staal(460-500 J/kgK). Het is onduidelijk welke het was, hoewel warmteverlies de berekende soortelijke warmte lager maakt dan de echte, dus het is waarschijnlijker dat het materiaal ijzer of staal is. 
Nog steeds is dit maar een schatting, want de soortelijke warmte wordt 2.3% (van de berekende soortelijke warmte) hoger voor elke kelvin die de eindtemperatuur hoger wordt. Er is waarschijnlijk veel warmte verloren gegaan aan de lucht, dus dat kan de waarde sterk beinvloeden.

Jammer genoeg was er geen tijd om de metingen van andere groepen te noteren.